archlinuxcn/lceda-pro/LICENSE

```diff
@@ -1,40 +1,224 @@
-一、用户注册
-　　（1） 用户资格：嘉立创EDA用户必须是适用法律下能够签订具有法律约束力的合同的个人，同时，我们的服务不向16周岁以下或被临时或无限期中止的嘉立创EDA用户提供。如您不符合条件，请不要使用我们的服务。您的嘉立创EDA帐户不得向其他方转让或出售。嘉立创EDA保留根据其意愿中止或终止您的帐户的权利。
-　　（2） 资料填写：为确保交易的正常进行，用户必须向嘉立创EDA提供真实、有效的个人资料，如个人资料有任何变动，必须及时通知嘉立创EDA更新。嘉立创EDA用户同意接收来自嘉立创EDA或者嘉立创EDA合作伙伴发出的邮件、信息。
-二、 隐私
-　　嘉立创EDA设有适用于所有用户并纳入用户协议的隐私声明。您应当在注册时阅读并接受隐私声明以使用嘉立创EDA网站，且您在作为嘉立创EDA用户期间将继续受其规定（及嘉立创EDA对隐私声明作出的任何修订）的约束。
-　　（1）个人数据资料。包括个人识别资料：如姓名、电话、微信、QQ、电子邮件地址等情况；个人背景：职业、教育程度、企业信息、学校信息等。在未经您同意及确认之前，嘉立创EDA不会将您的资料用于其它目的。
-　　（2）使用非个人信息。嘉立创EDA将通过您的IP地址来收集非个人化的信息，包括您的浏览器性质、操作系统种类、给您提供接入服务的ISP的域名等，以优化在您计算机屏幕上显示的页面。通过收集上述信息，我们亦进行客流量统计，从而改进网站的管理和服务。
-　　（3）个人数据安全。嘉立创EDA采用安全性的服务器使用控制设备保护您的个人数据。
-　　（4）个人数据披露和限制利用。当国家机构依照法定程序要求嘉立创EDA披露个人资料时，嘉立创EDA将根据执法单位的要求或为公共安全目的提供个人资料。在此情况下披露的任何内容，嘉立创EDA不承担任何责任。嘉立创EDA在符合下列条件时，对个人资料进行必要范围以外利用：已取得您的书面同意；为免除您在生命、身体或财产方面之急迫危险；为防止他人权益之重大危害；为增进公共利益，且无害于您的重大利益。
-（5）嘉立创EDA在线版账号或团队的元件库或模块库属于公开状态，任意用户可以搜索到并使用（包括收藏，二次编辑，克隆，直接使用等）。
-三、交易
-　　用户在嘉立创EDA进行物品购买时必须遵守以下条款：
-　　（1）您应就您使用嘉立创EDA的服务及对物品购买遵守所有适用的中国法律、法规、条例和地方性法律要求。您还必须确保您遵守本用户协议和隐私声明及纳入上述文件的所有其它条款和规则的所有规定。
-　　（2）如您在嘉立创EDA购买物品，您有义务完成与嘉立创EDA的交易，但法律或本用户协议禁止的交易除外。通过对一项物品下订单，您同意受该物品描述所含的出售条件的约束，只要该等出售条件不违反本协议或不是非法的。
-　　（3）嘉立创EDA网站上的广告，价目表和声明并不构成要约。嘉立创EDA有权在发现嘉立创EDA网站上显现的产品及订单的明显错误或缺货的情况下，单方面撤回该信息或撤销合同。嘉立创EDA保留对产品订购的数量的限制权。在下订单的同时，您也同时承认了您已经达到购买这些产品的法定年龄，并将对您在订单中提供的所有信息的真实性负责。
-　　（4）产品价格和可获性都在网站上指明。这类信息将随时更改且不提前发任何通知。送货费将另外结算，费用根据您选择的送货方式的不同而异。如果在确认了您的订单后，由于供应商提价，税额变化等引起的价格变化，或是由于网站的错误等造成价格错误的，嘉立创EDA会通过email或电话通知您。如您拒绝变更后的价格有权取消订单，如您同意按照变更后的价格继续交易的，您同意补足差价，嘉立创EDA将在您补足差价后向您提供有关产品。
-　　（5）嘉立创EDA标准版和专业版基础功能免费（包括但不限于编辑器，个人中心，工作区基础功能），任何个人、教育机构或企业用户均可以免费使用；企业增值服务会按定价收取费用（包括但不限于企业私有部署、技术培训、企业版专属功能等）。
-四、物品质量与售后服务
-　　（1）嘉立创EDA不提供无条件退换货服务，详见售后服务条款
-　　（2）物品的质量，有国家标准或专业标准的，按国家标准或专业标准执行；无前述标准的，按生产厂的企业标准执行；无生产厂企业标准的，由用户与嘉立创EDA协商确定。
-五、网站安全
-　　您同意，未经嘉立创EDA明示书面准许，您不会为了任何目的使用任何机器人、蜘蛛软件、刷屏软件或其他自动方式进入网站。此外，您同意您将不会：
-　　（1）进行任何对嘉立创EDA网站造成或可能造成（按嘉立创EDA自行酌情确定）不合理或不合比例的重大负荷的行为；
-　　（2）未经嘉立创EDA事先明示书面准许，对网站的任何内容（除您的个人信息以外）制作拷贝、进行复制、修改、制作衍生作品、分发或公开展示；
-　　（3）干扰或试图干扰网站的正常工作或网站上进行的任何活动；
-　　（4）在网站中使用违反适用的法律法规下可能视为被禁止或可能被禁止的任何内容；
-　　（5）对网站使用包含可能破坏、改变、删除、不利影响、秘密截取、未经授权而接触或征用任何系统、数据或个人资料的任何病毒、特洛依木马、蠕虫、定时炸弹、删除蝇、复活节彩蛋、间谍软件或其他电脑程序。
-六、知识产权声明
-　　（1）嘉立创EDA所提供的网络服务的相关著作权、专利权、商标、商业秘密及其它任何所有权或权利，均属嘉立创EDA所有。非经嘉立创EDA的同意，任何人或用户均不得擅自下载、复制、传输、改编、编辑，否则应负所有法律责任。
-（2）嘉立创EDA拥有向用户提供网络服务过程中所产生并储存于服务器中的任何数据信息的所有权。
-（3）任何人或企业未经嘉立创EDA书面许可，不得对嘉立创EDA以及嘉立创EDA官方库文件进行二次改造、销售，或包含在对外销售的数据包中。
-七、免责声明
-　　（1）嘉立创EDA有义务在技术上确保网站的正常运行，尽力避免服务中断或将中断时间限制在最短时间内，保证用户网上交易活动的顺利进行。但如因不可抗力或其他嘉立创EDA无法控制的原因使嘉立创EDA网站崩溃或无法正常使用导致网上交易无法完成或丢失有关的信息、记录等，嘉立创EDA不承担责任。
-（2）任何用户向嘉立创EDA提供错误、不完整、不实信息等造成不能正常使用嘉立创EDA服务或遭受任何其他损失，概与嘉立创EDA无关。
-（3）嘉立创EDA有义务确保嘉立创EDA编辑器运行正确或嘉立创EDA官方库的正确，但因未知的缺陷或官方库错误导致的生产损失，嘉立创EDA不承担责任。
-八、其它
-　　本协议在所有方面均受中华人民共和国法律管辖。本网站所有用户均不可撤销地受嘉立创EDA持有人所在地的中国深圳市福田区人民法院及上级法院的管辖。本协议的规定是可分割的，如本协议任何规定被裁定为无效或不可执行，该规定可被删除而其余条款应予以执行。您同意，在发生并购，本协议和所有纳入的协议可由嘉立创EDA自行酌情决定向第三方自动转让。标题仅为参考之用，在任何方面均不界定、限制、解释或描述该条的范围或限度。
-
-最后更新：2022.04.12
+# 嘉立创EDA用户协议
+
```

Comment:
1. Please add the distribution-allowed documentation to this repository, without writing it to the /usr/share/licenses directory.
2. It is recommended to download the LICENSE file from the official website, without manually updating it every time. [See this practice](https://gitlab.archlinux.org/archlinux/packaging/packages/discord/-/blob/main/PKGBUILD?ref_type=heads#L19).